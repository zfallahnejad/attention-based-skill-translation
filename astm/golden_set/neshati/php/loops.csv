118068,loops
1491895,loops
324584,loops
889949,loops
50079,loops
541091,loops
250259,loops
507674,loops
1226894,loops
862594,loops
218196,loops
1438393,loops
574632,loops
2844319,loops
238978,loops
31671,loops
1003917,loops
