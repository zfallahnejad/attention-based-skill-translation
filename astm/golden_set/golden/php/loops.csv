118068,26,36,0.41935483870967744
1491895,20,25,0.4444444444444444
324584,15,12,0.5555555555555556
50079,15,7,0.6818181818181818
889949,15,11,0.5769230769230769
541091,13,3,0.8125
250259,13,11,0.5416666666666666
1226894,12,8,0.6
507674,12,17,0.41379310344827586
862594,12,7,0.631578947368421
218196,11,12,0.4782608695652174
1438393,11,5,0.6875
574632,8,7,0.5333333333333333
2844319,8,7,0.5333333333333333
238978,8,5,0.6153846153846154
31671,7,7,0.5
1003917,7,8,0.4666666666666667
